plat/arm/board/rdn1edge/fdts/rdn1edge_nt_fw_config.dts

```diff
@@ -7,7 +7,7 @@
 /dts-v1/;
 / {
 	/* compatible string */
-	compatible = "arm,sgi-clark";
+	compatible = "arm,rd-n1e1edge";
```

Comment:
`rd-n1e1edge`
Is this intentional? It mentions both "n1" and "e1". Seems not consistent with the change you did in the rde1edge dts file.

Reply:
It was not intentional. Submitted a new patchset which changes the compatible string to 'rd-n1edge' and also updates the copyright headers